templates/cloudflare-pages/server.ts

```diff
@@ -8,6 +8,6 @@ if (process.env.NODE_ENV === "development") {
 
 export const onRequest = createPagesFunctionHandler({
   build,
-  getLoadContext: (context) => ({ env: context.env }),
+  getLoadContext: (context) => ({ env: context.env, cf: context.request?.cf }),
```

Comment:
Is waitUntil already provided to `context` here? It would be great to also pass that function by default

Reply:
I don't think so, but will take a look. And I'll look at the Vite template as well.